src/reducers/__tests__/deleteThought.ts

```diff
@@ -80,3 +80,20 @@ it('update cursor after thought deletion', () => {
     },
   ])
 })
+it('deletion of empty thought', () => {
+  const steps = [newThought(''), newThought('')]
+
+  const state = initialState()
+  const stateNew = reducerFlow(steps)(state)
+  const rootChildren = stateNew.thoughts.thoughtIndex[HOME_TOKEN].children
+
+  // root children before deletion
+  expect(rootChildren.length).toBe(2)
+
+  const stateAfterDeletion = reducerFlow([deleteEmptyThought])(stateNew)
+
+  // root children after deletion
+  const rootChildrenAfterDeletion = stateAfterDeletion.thoughts.thoughtIndex[HOME_TOKEN].children
+
+  expect(rootChildrenAfterDeletion.length).toBe(1)
```

Comment:
This only tests the number of children. We want to test that the specific empty thought was deleted. 

This makes a difference if, for example, the empty thoughts had different children.